### PR TITLE
Fixes: adding SearchResults Entity

### PR DESCRIPTION
**Pull request by maiorrulz**

### CODE REVIEW

File: Integrations/integration-RsaNetWitnessPacketsAndLogs.yml [hunk 1]
```diff
@@ -99,15 +99,18 @@ script:
         } else if ((typeof data) ==='object') {
             extractFromData(data);
         }
-        //May be empty elements which should be filtred
         queryResults = queryResults.filter(function (element) {
             return element != undefined || element != null;
         });
 
         if(queryResults.length > 0) {
+        	if(isEvent) {
```
Comment (anara123) [inline Integrations/integration-RsaNetWitnessPacketsAndLogs.yml]: Bad indentation
image

File: Integrations/integration-RsaNetWitnessPacketsAndLogs.yml [hunk 2]
```diff
@@ -99,14 +99,17 @@ script:
...
             extractFromData(data);
         }
-        //May be empty elements which should be filtred
         queryResults = queryResults.filter(function (element) {
             return element != undefined || element != null;
         });
 
         if(queryResults.length > 0) {
+            if(isEvent) {
+                    retVal.context.NetWitness.Events = queryResults;
```
Comment (anara123) [inline Integrations/integration-RsaNetWitnessPacketsAndLogs.yml]: bad indentation
image

File: Integrations/integration-RsaNetWitnessPacketsAndLogs.yml [hunk 3]
```diff
@@ -48,426 +48,456 @@ configuration:
...
-        if (Array.isArray(data)) {
-            data.forEach(function (element) {
-                extractFromData(element);
-            });
-        } else if ((typeof data) ==='object') {
-            extractFromData(data);
+
+        var isObjectEmpty = function (obj) {
+            return Object.keys(obj).length === 0;
         }
```
Comment (anara123) [inline Integrations/integration-RsaNetWitnessPacketsAndLogs.yml]: missing `;`

**Comment by maiorrulz** [comment]
Micron - Create RSA Netwitness script that accept time range #7509

**Pull request merged**